dask_jobqueue/core.py

```diff
@@ -210,12 +218,16 @@ def job_file(self):
                 f.write(self.job_script())
             yield fn
 
+    def submit_job(self, script_filename):
+        """ Sumbits job """
```

Comment:
Personally I am in favour of not having a docstring for this function since this is just a one-liner. More generally I think if your docstring is just repeating the function name, remove it.